.rubocop.yml

```diff
@@ -30,6 +30,7 @@ Naming/FileName:
 Rails/Output:
   Exclude:
     - lib/rmt/cli/**/*
+    - lib/rmt/import_script.rb
```

Comment:
`import_script` sounds to generic. Rename the script to `smt_import`?  You could also drop it into `cli` dir -- that's where all the stuff that prints to console lives. :grimacing: